output/standard.go

```diff
@@ -159,3 +163,37 @@ func (s *Standard) outputTrace(results []CheckResult, colorizer aurora.Aurora) {
 		}
 	}
 }
+
+// outputs results as a report - similar to OPA test output
+func (s *Standard) Report(results []*tester.Result) error {
+	reporter := tester.PrettyReporter{
+		Verbose:     true,
+		Output:      os.Stdout,
+		FailureLine: true}
+
+	dup := make(chan *tester.Result)
+
+	go func() {
```

Comment:
I don't see the purpose of using a channel here, since the goroutine just iterates over the results so everything is still processed serially. I think this should be removed.

Reply:
agree. fixed it.